README.md

```diff
@@ -30,7 +30,7 @@ the Cardano Shelley implementation:
   This document is a technical specification of the networking protocol.  It
   includes serialisation formats, necessary details of multiplexer and
   technical specifications of mini-protocols used by either _node-to-node_ and
-  _node-to-client_ flavors of the protocol. 
+  _node-to-client_ flavours of the protocol.
```

Comment:
IOHK style guide says to favor (:grin:) US spellings. I'm mostly pointing that out just to tease --- adhering to that in this repo would require a big diff.

Reply:
Thanks, I wasn't aware of that.  Do you have a link to that style guide?

Reply:
I've sent you a Google Docs like on Slack.